app/models/BannerDesign.scala

```diff
@@ -41,6 +41,8 @@ sealed trait BannerDesignVisual
 object BannerDesignVisual {
   case class Image(
       kind: String = "Image",
+      style: Option[String],
+      colourTheme: Option[String],
```

Comment:
instead of defining these here and in ChoiceCards, add them to the BannerDesign model.